src/render/tutorials/getting-started-with-docker/2_0.html.md

```diff
@@ -33,120 +33,164 @@ Project has the following structure:
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Knot.x</title>
-  <link href="https://bootswatch.com/4/superhero/bootstrap.min.css" rel="stylesheet"/>
-</head>
-<body>
-<div class="container-fluid">
-  <div class="row">
-    <div class="col-sm-12">
-      <div class="panel panel-default">
-        <div class="panel-heading">Books list</div>
-        <div class="panel-body">
-          This page lists Java related books provided by <a
-            href="https://www.googleapis.com/books/v1/volumes?q=java">Googleapis book service</a>.
-        </div>
-      </div>
-    </div>
-  </div>
-  <div class="row">
-   <knotx:snippet data-knotx-task="bookslist">
-      {{#each books._result.items}}
-        <div class="col-sm-3">
-        <div class="card">
-          <img class="card-img-top"
-             src="{{volumeInfo.imageLinks.thumbnail}}"
-             alt="{{volumeInfo.title}}">
-          <div class="card-block">
-            <h4 class="card-title">{{volumeInfo.title}}</h4>
-            <p class="card-text">
-              {{#each volumeInfo.authors as |author|}}
-                {{author}}{{#unless @last}}, {{/unless}}
-              {{/each}}<br />
-              Published: {{volumeInfo.publishedDate}}
-            </p>
-          </div>
-         </div>
-      </div>
-      {{/each}}
-    </knotx:snippet>
-
-    </div>
-</div>
-</body>
-</html>
-```
-
-### Templates repository configuration
-We will use [fsRepoConnectorHandler](https://github.com/Knotx/knotx-repository-connector/tree/master/fs), its purposes are purely academical.
-It is not designed to be used as a production ready solution (you should use [httpRepoConnectorHandler](https://github.com/Knotx/knotx-repository-connector/tree/master/http) there).
-
-Create `knotx/conf/routes/handlers/fsRepoConnectorHandler.conf` file with the following:
-```hocon
-# Path to the directory on the local filesystem which will be the root for requested templates
-catalogue = "/usr/local/knotx/repository/"
-```
-This way we define the `repository` directory that will be our files content repository.
+## Create a new routing path
 
-### openapi.yml
-Open `knotx/conf/openapi.yml` and add following path definition:
+Let's take a look at `knotx/conf/openapi.yaml` file.
 
-```
-  /content/*:
+It's an [OpenAPI 3.0](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md) specification file that exposes APIs.
+
+> In OpenAPI terms, paths are endpoints (resources), such as /users or /reports/summary/, that your API exposes, and operations are the HTTP methods used to manipulate these paths, such as GET, POST or DELETE. [~source](https://swagger.io/docs/specification/paths-and-operations/))
+
+We can see three distinguishable parts of this file:
+
+ 1.  Information about our project, which is quite self-explanatory:
+ 
+      ```
+      ...
+      
+      openapi: "3.0.0"
+      info:
+        version: 1.0.0
```

Comment:
This has no quotation marks while `openapi` has them. Are they optional or is there a greater reason? Might be not obvious/confusing

Reply:
Maybe we can add also:
```
  "contact": {
    "name": "Knot.x Community",
    "url": "http://knotx.io/community/"
  },
```

Reply:
It looks like the quotation marks are not mandatory.
I've copied this part from Starter Kit, so that might be a good idea to change that as well in other files.
What do you think @tomaszmichalak ?